doc/upgrade.md

```diff
@@ -52,6 +52,16 @@ You can also use Tenderly to manually submit contract code for [private verifica
 
 If you see an error related to multiple SPDX license identifiers, remove all SPDX license identifiers from `flattened.sol` except for a single one.
 
+## Task
+
+Alternatively, you can also run the `etherscan-verify-deployments` task from this repository available as a `yarn` script:
```

Comment:
Any reason to list this as an alternative vs. replacing the docs about running `npx hardhat verify` with the docs here? If the former is the easiest way to verify the source code of all contracts I think it makes sense to just highlight that as the primary method.

Reply:
OK, will reorganize!